packages/@ourworldindata/grapher/src/entitySelector/EntitySelector.tsx

```diff
@@ -591,7 +602,11 @@ export class EntitySelector extends React.Component<{
             this.selectionArray.toggleSelection(entityName)
         } else {
             this.selectionArray.setSelectedEntities([entityName])
-            if (this.props.onDismiss) this.props.onDismiss()
+            if (this.props.onDismiss) {
+                this.dismissTimer = setTimeout(() => {
+                    if (this.props.onDismiss) this.props.onDismiss()
+                }, 250)
```

Comment:
I see Marwa's comment that was suggesting this, but I think this now feels even weirder than it did before.
You can just about glimpse that something is animating, but then it the modal just closes on you without notice, and that feels quite broken.